### PR TITLE
Ado 82740

### PR DESCRIPTION
**Pull request by alexunique0519**
### Description

Update Overview page: "What to know before you begin" to "what you'll need", and change texts and translations to follow: 

image
image

### What to test for/How to test
pull code and test in browser
### Additional Notes
n/a

### CODE REVIEW

**Review by alex-solo — CHANGES_REQUESTED** [review]
Just the period in `youNeedEndingText`  of the French translation is missing a period at the end

File: pages/index.tsx [hunk 1]
```diff
@@ -101,7 +98,9 @@ const Home: NextPage<{ adobeAnalyticsUrl: string }> = ({
             <li dangerouslySetInnerHTML={{ __html: tsln.maritalStatusText }} />
             <li dangerouslySetInnerHTML={{ __html: tsln.partnerText }} />
           </ul>
-
+          <p className="xs:mt-0 sm:mt-8 xs:pr-3 w-full sm:w-[73%]">
```
Comment (MarcoGoC) [inline pages/index.tsx:101]: 73% ???

**Review by MarcoGoC — APPROVED** [review]
except for the css comment it looks good

File: pages/index.tsx (hunk 1 above)
Comment (alexunique0519) [reply pages/index.tsx:101]: If you look at the figma mockup you will know, it wraps the last word to the next line.

**Review by alex-solo — APPROVED** [review]

File: pages/index.tsx (hunk 1 above)
Comment (MarcoGoC) [reply pages/index.tsx:101]: would not be better if the figma follows the same size as the current site?

**Pull request merged**